packages/expo-dev-menu/app/DevMenuWebBrowser.ts

```diff
@@ -0,0 +1,91 @@
+import { NativeModules, AppState, Linking } from 'react-native';
+
+const DevMenu = NativeModules.ExpoDevMenuInternal;
+
+let _redirectHandler: ((event: any) => void) | null = null;
+let _onWebBrowserCloseAndroid: null | (() => void) = null;
+let _isAppStateAvailable: boolean = AppState.currentState !== null;
+
+function _onAppStateChangeAndroid(state: any) {
```

Comment:
I dislike the idea of prefixing private functions with `_` 😅 If it's not exported then it's private 🤷‍♂️ Anyway, let's do as you think is best 😉